tools/cpudist.py

```diff
@@ -208,7 +208,7 @@
 max_pid = int(open("/proc/sys/kernel/pid_max").read())
 
 b = BPF(text=bpf_text, cflags=["-DMAX_PID=%d" % max_pid])
-b.attach_kprobe(event_re="^finish_task_switch$|^finish_task_switch\.isra\.\d$",
+b.attach_kprobe(event_re="^finish_task_switch$|^finish_task_switch\\.isra\\.\\d$",
```

Comment:
Looks correct according to https://github.com/python/cpython/pull/99011, though I don't get this warning with Python 3.10 on Ubuntu.

Could we use raw string here, for both Python 2 and Python 3 ?  `\\` looks ugly.

Reply:
I actually considered raw strings first but figured proper escapes were more idiomatic.. 😅
But ok - I will convert the regexps to raw strings if you prefer. Thanks for taking a look!

Reply:
Update pushed.